app/src/main/java/woowacourse/shopping/feature/cart/CartViewModel.kt

```
@@ -200,81 +164,97 @@ class CartViewModel(
-        cartRepository.updateQuantity(cartItem.id, CartQuantity(cartItem.quantity + 1), {
-            updateCartQuantity()
-            selectedCartMap[cartItem.id]?.quantity = (selectedCartMap[cartItem.id]?.quantity ?: 0) + 1
-        }, {
-            _event.postValue(CartUiEvent.ShowToast(ToastMessageKey.FAIL_INCREASE))
-        })
+        viewModelScope.launch {
+            try {
+                cartRepository.updateQuantity(cartItem.id, CartQuantity(cartItem.quantity + 1))
+                updateCartQuantity()
+                _selectedCartMap[cartItem.id]?.quantity = (_selectedCartMap[cartItem.id]?.quantity ?: 0) + 1
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)